internal/guid/guid.go

```diff
@@ -17,6 +17,11 @@ func New() GUID {
 	return g
 }
 
+func Empty() GUID {

```

Comment:
What is the point of this function? Anyone who wants an empty guid can just do `guid.GUID{}`. Why call a function to do this?

Reply:
Removed.